### DISCUSSION TITLE
Separate Server

### DISCUSSION THREAD

**Discussion by leadhkr** [jaredpalmer/razzle#1586] [Q&A]
## ❓Question

@jaredpalmer awesome job, razzle looks great. I'm in the midst of rethinking my nextjs application and migrating to razzle. If I have a django -based rest API (my backend server, which handles just about everything -- auth, db access, etc), where would it be best to make those calls from my razzle application? Prior to migrating to nextjs (cra), I was making these calls using react-saga. Curious what your take is on this.

Thanks!

**Comment by fivethreeo** [comment]
This is really not a razzle issue, will move this to discussions
